apps/teams-test-app/src/components/SearchAPIs.tsx

```diff
@@ -0,0 +1,42 @@
+import { search } from '@microsoft/teams-js';
+import React, { ReactElement } from 'react';
+
+//import { search } from '../../../../packages/teams-js/src/public/search';
```

Comment:
We should not merge code that's commented out. If it's needed, it shouldn't be commented out. And if it's not needed, please remove it completely.